cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/lib/util/golang.sh

```diff
@@ -8,7 +8,7 @@ function os::golang::verify_go_version() {
 
 	local go_version
 	go_version=($(go version))
-	if [[ "${go_version[2]}" != go1.7* ]]; then
+	if [[ "${go_version[2]}" != go1.8* ]]; then
```

Comment:
Also disregard my comment about a common utility. I misread the diff and thought all these changes were in Origin. NACK to changing these vendored files - we should let them change as we rebase newer versions of the dependencies.